bindata/network/ovn-kubernetes/common/alert-rules.yaml

```diff
@@ -62,3 +62,12 @@ spec:
       for: 15m
       labels:
         severity: warning
+    - alert: OVNKubernetesResourceRetryFailure
+      annotations:
+        summary: OVN Kubernetes experiencing Kubernetes Resource Retry Failure at an elevated rate.
+        description: OVN Kubernetes experiencing Kubernetes resource processing failure even
+         after reaching its maximum retry limit and was no longer processed.
+      expr: rate(ovnkube_resource_retry_failures_total[1m]) > 0
+      for: 10m
+      labels:
+        severity: warning
```

Comment:
I think this is critical. WDYT?

Reply:
yes, it must be critical. changed it.

Reply:
changing `serverity` back to `warning`, as this may trigger for a case when object's spec is invalid (for example, invalid IP address specified on the egress IP object). and making this as critical alert is too much (as per description https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md#critical-alerts).

Reply:
Agreed. Thank you peri!